src/persist/src/operators/upsert.rs

```diff
@@ -64,7 +67,10 @@ pub trait PersistentUpsert<G, K: Codec, V: Codec, T> {
         name: &str,
         as_of_frontier: Antichain<u64>,
         persist_config: PersistentUpsertConfig<K, V>,
-    ) -> Stream<G, ((K, V), u64, isize)>
+    ) -> (
+        Stream<G, ((K, V), u64, isize)>,
```

Comment:
I thought we were moving the the single stream of Result instead of the dual stream. Any context here on why this is the old way?

Reply:
I'm somewhat torn on this one: I'd like returning a single stream but I hesitated here because of how we can just pass updates through here, when we don't wrap: https://github.com/MaterializeInc/materialize/blob/main/src/persist/src/operators/stream.rs#L103

If we want to return a single result stream I'd have to wrap every incoming update in an `Ok()`. It might be that I'm too naive and not trusting the compiler enough to optimize this away.

Reply:
Okay, that's good enough for me. Let's add a comment to that effect around here somewhere